src/main/java/io/github/moulberry/notenoughupdates/overlays/TimersOverlay.java

```diff
@@ -130,6 +130,7 @@ protected Vector2f getSize(List<String> strings) {
 	private static final ItemStack COMMISSIONS_ICON = new ItemStack(Items.iron_pickaxe);
 	private static final ItemStack EXPERIMENTS_ICON = new ItemStack(Items.enchanted_book);
 	private static final ItemStack COOKIE_ICON = new ItemStack(Items.cookie);
+	private static final ItemStack QUEST_ICON = new ItemStack(Items.book);
```

Comment:
Can we make this like a sign? iirc its like a sign board in the world 
Only because book is alr taken

Reply:
Done

![image](https://user-images.githubusercontent.com/52578495/196727568-0230d1e2-5d75-4c04-8293-95ee87e22db1.png)